content/c-sharp/concepts/arrays/terms/reverse/reverse.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: '.Reverse()'
+Description: 'Reverses the sequence of a subset of the elements in the one-dimensional Array.'
+Subjects: 
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arrays'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/build-web-apps-with-asp-net'
+---
+
+## Syntax
+
+```pseudo
+// Reverses the sequence of a subset of the elements in the one-dimensional Array.
+
+public static void **Reverse** (Array array, int index, int length); 
```

Comment:
In docs we generally show the method in use in a general case:
`result = Array.Reverse(sourceArray, index, length)`

Then we have a sentence describing what the elements `result`, `sourceArray`, `index` and `length` are and what they do. You also should mention what parameters are optional and what happens when they're omitted.